### PR TITLE
Open telemetry samples for v9

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: samples/open-telemetry/application-insights/Core_9/Endpoint/Endpoint.csproj [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net8.0</TargetFramework>
+		<LangVersion>12.0</LangVersion>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.*" />
+		<PackageReference Include="Azure.Monitor.OpenTelemetry.Exporter" Version="1.1.0" />
```
Comment (kentdr) [inline samples/open-telemetry/application-insights/Core_9/Endpoint/Endpoint.csproj:11]: Should this be a wildcard? `1.*`?

File: samples/open-telemetry/jaeger/Core_8/Publisher/Publisher.csproj [hunk 2]
```diff
@@ -8,8 +8,7 @@
 
 	<ItemGroup>
 		<PackageReference Include="NServiceBus" Version="8.*" />
-		<PackageReference Include="OpenTelemetry" Version="1.*" />
-		<PackageReference Include="OpenTelemetry.Exporter.Jaeger" Version="1.*" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.7.0" />
```
Comment (kentdr) [inline samples/open-telemetry/jaeger/Core_8/Publisher/Publisher.csproj:11]:
```suggestion
    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.*" />
```
Should this be a wildcard as well?

File: samples/open-telemetry/jaeger/Core_9/Publisher/Publisher.csproj [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net8.0</TargetFramework>
+		<LangVersion>12.0</LangVersion>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="NServiceBus" Version="9.0.0-alpha.10" />
+		<PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.7.0" />
```
Comment (kentdr) [inline samples/open-telemetry/jaeger/Core_9/Publisher/Publisher.csproj:11]:
```suggestion
		<PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.*" />
```
Should this be a wildcard reference?

File: samples/open-telemetry/jaeger/Core_9/Subscriber/Subscriber.csproj [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <LangVersion>12.0</LangVersion>
+  </PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="NServiceBus" Version="9.0.0-alpha.10" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.7.0" />
```
Comment (kentdr) [inline samples/open-telemetry/jaeger/Core_9/Subscriber/Subscriber.csproj:11]:
```suggestion
    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.*" />
```
Should this be a wildcard reference?

**Review by bording — APPROVED** [review]

**Pull request merged**